headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseItemResourceImpl.java

```diff
@@ -232,51 +304,6 @@ public WarehouseItem postWarehousByExternalReferenceCodeWarehouseItem(
 					getCommerceInventoryWarehouseItemId()));
 	}
 
-	@Override
-	public WarehouseItem postWarehouseItemByExternalReferenceCode(
```

Comment:
@marco-leo also about this I'm not 100% sure, but in the end I think the use case is well covered with other methods, plus this was very tricky and confusing.. Let me know what do you think about this. Thank you!